src/core/scenario.jl

```diff
@@ -337,7 +341,7 @@ function Scenario(d::Dict; flex_hvac_from_json=false)
             end
             chiller_inputs = merge(chiller_inputs, dictkeys_tosymbols(d["ExistingChiller"]))
         else
-            chiller_inputs[:cop] = 1.0
+            chiller_inputs[:cop] = cooling_load.existing_chiller_cop
```

Comment:
@zolanaj This "fix" breaks the thermal storage test results because it was using a chiller cop of 1.0 before when ExistingChiller was not in the inputs. I don't know why it was set to 1.0 before. I just wanted to have you double check this since it breaks the test you wrote. If you agree, we can just adjust the expected values.

Reply:
@Bill-Becker I agree with this update, as well as the required changes to expected test values.  Full review to follow.